lib/versioning/conan/index.ts

```diff
@@ -0,0 +1,613 @@
+import * as semver from 'semver';
+import { SemVer, parseRange } from 'semver-utils';
+import { logger } from '../../logger';
+import { regEx } from '../../util/regex';
+import { api as looseAPI } from '../loose';
+import type { NewValueConfig, VersioningApi } from '../types';
+
+export const id = 'conan';
+export const displayName = 'conan';
+export const urls = [
+  'https://semver.org/',
+  'https://github.com/podhmo/python-node-semver',
+  'https://github.com/podhmo/python-node-semver/tree/master/examples',
+  'https://docs.conan.io/en/latest/versioning/version_ranges.html#version-ranges',
+];
+export const supportsRanges = true;
+export const supportedRangeStrategies = ['auto', 'bump', 'widen', 'replace'];
+
+const MIN = 1;
+const MAX = -1;
+
+export function cleanVersion(version: string): string {
+  if (version) {
+    return version
+      .replace(/,|\[|\]|"|include_prerelease=|loose=|True|False/g, '')
+      .trim();
+  }
+  return version;
+}
+
+export const getOptions = (
+  input: string
+): { loose: boolean; includePrerelease: boolean } => {
+  let includePrerelease = false;
+  let loose = true;
+  if (input) {
+    includePrerelease =
+      input.includes('include_prerelease=True') &&
+      !input.includes('include_prerelease=False');
+    loose = input.includes('loose=True') || !input.includes('loose=False');
+  }
+  return { loose, includePrerelease };
+};
+
+export const makeVersion = (
+  version: string,
+  options: semver.Options
+): string | boolean => {
+  const splitVersion = version.split('.');
+  const prerelease = semver.prerelease(version, options);
+
+  if (prerelease && !options.includePrerelease) {
+    if (!Number.isNaN(+prerelease[0])) {
+      const stringVersion = `${splitVersion[0]}.${splitVersion[1]}.${splitVersion[2]}`;
+      return semver.valid(stringVersion, options);
+    }
+    return false;
+  }
+
+  if (
+    options.loose &&
+    !semver.valid(version, options) &&
+    splitVersion.length !== 3
+  ) {
+    return semver.valid(semver.coerce(version, options), options);
+  }
+
+  return semver.valid(version, options);
+};
+
+export const isVersion = (input: string): string | boolean => {
```

Comment:
Please do not export all functions, only do that if they are used by datasource or manager.

so only export metadata and `api` below.